backend/app/routers/visualization.py

```diff
@@ -0,0 +1,85 @@
+import mimetypes
+
+from beanie import PydanticObjectId
+from fastapi import APIRouter, HTTPException, Depends, Security
+from fastapi import File, UploadFile, Response, Request, Form
+
+from minio import Minio
+
+from app import dependencies
+from app.config import settings
+from app.keycloak_auth import get_current_user
+
+from app.models.visualization import VisualizationOut, VisualizationIn, VisualizationDB
+from app.routers.utils import get_content_type
+
+router = APIRouter()
+
+
+@router.post("/name={name}&description={description}", response_model=VisualizationOut)
```

Comment:
in a post endpoint maybe it's better to pass in the variables as body instead of query parameters? Thoughts?

Reply:
Could you try remove name={name}&description={description} in the path and see if fastAPI can still pick it up as query paratmers